.gitlab-ci/packet.yml

```diff
@@ -242,10 +236,6 @@ packet_ubuntu20-calico-ha-recover-noquorum:
     RECOVER_CONTROL_PLANE_TEST: "true"
     RECOVER_CONTROL_PLANE_TEST_GROUPS: "etcd[1:]:kube_control_plane[1:]"
 
-packet_debian10-cilium-svc-proxy:
```

Comment:
is there an equivalent test with cilium-svc-proxy?

Reply:
Currently, no.

Do we need to reserve it and change to Debian 12?

Reply:
Not sure what svc-proxy means...

Reply:
Compared to the original cilium test.
```diff
---
# Instance settings
cloud_image: debian-12
- mode: default
+ mode: ha

# Kubespray settings
kube_network_plugin: cilium
+ enable_network_policy: true

+ cilium_kube_proxy_replacement: strict
```